com.examples.exercise/src/main/java/com/examples/exercise/SchoolController.java

```diff
@@ -0,0 +1,17 @@
+package com.examples.exercise;
+
+import java.util.List;
+
+public class SchoolController {
+	private Database database;
+
+	public SchoolController(Database database) {
+		super();
+		this.database = database;
+	}
+
+	public List<Student> getAllStudents() {
+		return database.getAllStudentsList();
+	}
+
```

Comment:
@LorenzoBettini I don't know what to do, they are present in my local copy of the branch, I can see and modify them from eclipse and from filesystem. In addition "git diff origin/master HEAD" only show I  added SchoolController.java and SchoolControllerTest.java and modified pom.xml